ci-test-files/centos8-imagestream.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   - annotations: null
     from:
       kind: DockerImage
-      name: quay.io/kubevirt/centos8-container-disk-images
+      name: quay.io/containerdisks/centos:8.4
     importPolicy:
       scheduled: true
     name: latest
```

Comment:
First, this is a test only file.

Also, I'm not sure it will work. I think that the `.name` field should be "8.4" and the `from.name` fiels should be "quay.io/containerdisks/centos", without the tag.

And HCO only supports image stream with `latest` tag.

Reply:
Thanks, I adjusted to `name:`.
@nunnatsa What do you mean by 

> And HCO only supports image stream with latest tag.

Does this mean that `quay.io/containerdisks/centos ` requires the `latest` tag?

Reply:
We're not using image stream for the community version. This file is only for CI testing, to test the image stream logic.

The actual DataImportCronTemplate files are using URLs.

Reply:
@dominikholler - please fix this file (cancel the changes) and add release notes so we can go on with this pr.